src/commands/create.js

```diff
@@ -82,6 +81,16 @@ class CreateCommand extends ApifyCommand {
 
         const packageJsonPath = path.join(actFolderDir, 'package.json');
         const requirementsTxtPath = path.join(actFolderDir, 'requirements.txt');
+        const readmePath = path.join(actFolderDir, 'README.md');
+
+        const localReadmeSuffix = await getLocalReadmeSuffix(manifestPromise);
+
+        // Add localReadmeSuffix which is fetched from manifest to README.md
+        // The suffix contains local development instructions
+        if (localReadmeSuffix && fs.existsSync(readmePath)) {
+            const readmeContent = fs.readFileSync(readmePath, 'utf8');
```

Comment:
It probably can't realistically fail here, but this is not the correct way of reading a file. You should try to read it right away and potentially handle the error. Not first check for its existence and then read it. It creates race conditions.